docs/UsersGuide/user/unit-testing.md

```diff
@@ -187,37 +187,44 @@ The F′ Prime build system provides targets for building and running
 component unit tests.
 
 To build unit tests, go to the component directory (not the *test/ut*
-directory) and run *make ut* to build with code coverage enabled, or run
-*make ut\_nocov* to build with code coverage disabled.
+directory) and run `fprime-util generate --ut`.
 
 To run unit tests, go to the component directory (not the *test/ut*
-directory) and run *make run\_ut* to run with coverage enabled, or *make
-run\_ut\_nocov* to run with coverage disabled.
-
-Components that call into libraries have two ways to write tests: i)
-link against the library in the test, or ii) link against a mock or stub
-library. Selecting from either of these two options depend on the
-circumstances. If selecting the link against the library in the test,
-avoid writing the mock or stub library. This approach proves that the
-component code works with the actual library. If selecting the link
-against a mock or stub library, it will make it easier to induce the
-behaviors for testing, such as injecting faults. This approach may be
+directory) and run `fprime-util check --<params>`. 
```

Comment:
Typically I like the unix style description of optional parameters:

```
fprime-util check [parameter flags]
```

Then make the table:

`--all`
`--coverage`
`--leak`